index.html

```diff
@@ -35,184 +32,163 @@
                         </ul>
-                </nav>
-                </label>
-            </div>
-        </div>
+                    </label>
+                </nav>     
     </header>
 
-    <!--Hero Page-->
-
-    <section>
-        <div class="background">
-            <div class="container">
-                <div class="maintext">
-                    <h1>Hey there. I’m <br>Lukman
-                        <span class="h1-highlight"><br>I’m a software <br>developer</span></h1>
-                    <div class="subtext">
-                        <p> I can help you build a product,
-                            feature or website. Look through some of my work
-                            and experience! If you like what you see and
-                            have a project you need coded,
-                            don’t hestiate to contact me.</p>
-                        <ul class="social">
-                            <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg"
-                                        alt="twitter"></a></li>
-                            <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
-                            <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
-                            <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a>
-                            </li>
-                            <li><a href="#"><img src="images\angellist.svg"></a></li>
-                        </ul>
-                    </div>
+    <!--Hero/Intro Page-->
 
+    <main class="background flex-container">
+        <ul class="header-desktop">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <h1 class='h1-top-space'>Hey there. I’m Lukman</h1>
+        <h1 class="h1-highlight">I’m a software developer</h1>
+            
+        <p class="hero-text">I can help you build a product,
+            feature or website. Look through some of my work
+            and experience! If you like what you see and
+            have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+        <ul class="social-icons">
+            <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
+            <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images\angellist.svg"></a></li>
+        </ul>
+    </main>
+
+    <!--Project Section-->
+
+    <section class="flex-container">
+        <h2>My Recent Works</h2>
+        <hr>
+        <article class="grid-container">
```

Comment:
- [ ] your project's cards design looks great, but it should have 2 rows with 3 projects on each row, currently, you have 1 row with three project, kindly add 3 more in another row below it to match the Figma design 
![image](https://user-images.githubusercontent.com/89282221/193097192-fc7c8f72-bd20-4373-b8aa-ba1dccb4d8a0.png)